github/codespaces_orgs.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2025 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// CodespacesOrgAccessControlRequest represent request for SetOrgAccessControl.
+type CodespacesOrgAccessControlRequest struct {
+	// Visibility represent which users can access codespaces in the organization.
+	// Can be one of: disabled, selected_members, all_members, all_members_and_outside_collaborators.
+	Visibility string `json:"visibility"`
+	// SelectedUsernames represent the usernames of the organization members who should have access to codespaces in the organization.
+	// Required when visibility is selected_members.
+	SelectedUsernames []string `json:"selected_usernames,omitempty"`
+}
+
+// ListInOrg lists the codespaces associated to a specified organization.
+//
+// GitHub API docs: https://docs.github.com/rest/codespaces/organizations#list-codespaces-for-the-organization
+//
+//meta:operation GET /orgs/{org}/codespaces
+func (s *CodespacesService) ListInOrg(ctx context.Context, org string, opts *ListOptions) (*ListCodespaces, *Response, error) {
```

Comment:
We already talked a lot about proper naming:
```suggestion
func (s *CodespacesService) ListOrganizationCodespaces(ctx context.Context, org string, opts *ListOptions) (*ListCodespaces, *Response, error) {
```